### PR TITLE
add RBD storage support

### PR DESCRIPTION
**Pull request by pixiake**
Add support for RBD storage.

### CODE REVIEW

File: pkg/apis/kubekey/v1alpha1/plugins_types.go [hunk 1]
```diff
@@ -2,17 +2,21 @@ package v1alpha1
 
 type Storage struct {
 	DefaultStorageClass string      `yaml:"defaultStorageClass" json:"defaultStorageClass,omitempty"`
-	LocalVolume         LocalVolume `yaml:"localVolume" json:"localVolume,omitempty"`
-	NfsClient           NfsClient   `yaml:"nfsClient" json:"nfsClient,omitempty"`
-	GlusterFS           GlusterFS   `yaml:"glusterFS" json:"glusterFS,omitempty"`
-	CephRBD             CephRBD     `yaml:"cephRBD" json:"cephRBD,omitempty"`
+	LocalVolume         LocalVolume `yaml:"local" json:"locaVolume,omitempty"`
+	NfsClient           NfsClient   `yaml:"nfs" json:"nfs,omitempty"`
```
Comment (rayzhou2017) [inline pkg/apis/kubekey/v1alpha1/plugins_types.go:6]: why here is nfsclient, while the yaml is nfs?

Comment (pixiake) [reply pkg/apis/kubekey/v1alpha1/plugins_types.go:6]: The name of this plugin is nfsclient.

File: pkg/apis/kubekey/v1alpha1/default.go [hunk 2]
```diff
@@ -126,3 +126,35 @@ func SetDefaultClusterCfg(cfg *ClusterSpec) Kubernetes {
...
+		cfg.Storage.NfsClient.Enabled = true
+	}
+	if cfg.Storage.CephRBD.StorageClassName != "" {
+		cfg.Storage.CephRBD.Enabled = true
+	}
+	if cfg.Storage.GlusterFS.StorageClassName != "" {
+		cfg.Storage.GlusterFS.Enabled = true
+	}
+
+	if cfg.Storage.DefaultStorageClass != "" {
```
Comment (min-zh) [inline pkg/apis/kubekey/v1alpha1/default.go:144]: if **cfg.storage.DefaultStorageClass** is none of the listed cases,  what will happen ?

File: pkg/apis/kubekey/v1alpha1/plugins_types.go (hunk 1 above)
Comment (min-zh) [reply pkg/apis/kubekey/v1alpha1/plugins_types.go:6]: [nfs](https://github.com/kubernetes-incubator/external-storage/tree/master/nfs) and [nfs-client](https://github.com/kubernetes-incubator/external-storage/tree/master/nfs-client) are two different **external-storage** of [kubernetes-incubator](https://github.com/kubernetes-incubator/external-storage) .  Actually what we use is nfs-client, i suggest use nfs-client for unity and no confoud.

File: pkg/apis/kubekey/v1alpha1/default.go (hunk 2 above)
Comment (pixiake) [reply pkg/apis/kubekey/v1alpha1/default.go:144]: No default storage classe will be set.

File: pkg/apis/kubekey/v1alpha1/plugins_types.go (hunk 1 above)
Comment (pixiake) [reply pkg/apis/kubekey/v1alpha1/plugins_types.go:6]: Yes. Then we'll still use `nfsClient`. @rayzhou2017 Does it need to be changed in readme？

Comment (rayzhou2017) [reply pkg/apis/kubekey/v1alpha1/plugins_types.go:6]: yes, if it is that case, then make it consistent.

**Pull request merged**